roles/metrics/tasks/main.yml

```diff
@@ -1,34 +1,34 @@
-# Setup our metrics services via docker-compose
-
 - name: Clone our metrics repo
   ansible.builtin.git:
     repo: https://github.com/planetary-social/metrics.git
-    dest: /home/{{ admin_username }}/metrics
+    dest: '{{ homedir }}/metrics'
+    version: 'canon'
```

Comment:
What is `canon`?

Reply:
Oh, copy/paste miss from the docs, sorry. I'll remove it.

Reply:
That's the main branch of the metrics repo.